### PR TITLE
Update README.md to add to samples search page

### PR DESCRIPTION
**Pull request by spboyer**

### CODE REVIEW

**Comment by spboyer** [comment]
adds template to https://learn.microsoft.com/en-us/samples/browse/?expanded=azure&languages=azdeveloper

File: README.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+---
+page_type: sample
+languages:
+- azdeveloper
+- javascript
+- bicep
+- typescript
```
Comment (paulyuk) [inline README.md:7]:
```suggestion
```

Comment (paulyuk) [reply README.md:7]: Like except it's not typescript

**Review by paulyuk — CHANGES_REQUESTED** [review]
LGTM with change

**Pull request merged**